src/middleware/storeLocals.js

```diff
@@ -0,0 +1,12 @@
+const storeLocals = (req, res, next) => {
```

Comment:
where do you use this file?

Reply:
No. Used it for ejs. Then remembered that we don't use it and didn't delete.
Will clean up